tools/link.py

```diff
@@ -1878,7 +1878,7 @@ def get_full_import_name(name):
   settings.PRE_JS_FILES = options.pre_js
   settings.POST_JS_FILES = options.post_js
 
-  settings.MINIFY_WHITESPACE = settings.OPT_LEVEL >= 2 and settings.DEBUG_LEVEL == 0 and not options.no_minify
+  settings.MINIFY_WHITESPACE = settings.OPT_LEVEL >= 2 and settings.DEBUG_LEVEL == 0 and not options.no_minify #shouldn't no_minify be equivalent to g1?
```

Comment:
Is this intentional?

Reply:
OK. I think the thing I saw was there were places that DEBUG_LEVEL is referred to where no_minify wasn't, and thought maybe this can be simplified (e.g. removing settings.no_minify and just making it equivalent to g1), but that's not for this PR in any case.